src/components/DocumentModals/ShowDocumentLocalModal.jsx

```diff
@@ -0,0 +1,54 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import ClearIcon from '@mui/icons-material/Clear';
+
+/**
+ * ShowDocumentsModal Component - Popup modal that shows client documents if gotten
+ * explicit permission via ACL
+ *
+ * @memberof DocumentModals
+ * @name ShowDocumentLocalModal
+ */
+
+const ShowDocumentLocalModal = ({ showModal, setShowModal, fileBlobUrl }) => (
+  <Dialog
+    fullScreen
+    open={showModal}
+    aria-labelledby="dialog-title"
+    onClose={() => setShowModal(false)}
+  >
+    <DialogTitle id="dialog-tile">Documents</DialogTitle>
+
+    <DialogContent
+      sx={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: 5
+      }}
+    >
+      <iframe src={fileBlobUrl} title="File Preview Local" height="100%" width="100%" />
```

Comment:
nitpick: this kicks rendering of the file to the browser. Probably fine, but not all browsers may be able to render the same file types. And certain ad blockers disable iframes like this.

If we get enough users for this to become an issue though, our app's in a good spot.